src/weatherreport/src/weatherreport.erl

```diff
@@ -0,0 +1,180 @@
+%% -------------------------------------------------------------------
+%%
+%% riaknostic - automated diagnostic tools for Riak
```

Comment:
We might be able to modify this to say that it's derived from riaknostic - clearly they don't copyright the entire file with CouchDB stuff in it :)

Reply:
Looking at `git blame`, this was originally implemented by Sean Cribbs at Basho, then later heavily modified by Mike Wallace and Paul Davis.

But, yeah, it's messy and I'm not sure the best approach for "remixes" like this.

Reply:
As it's not a copyright header, you can do what you want with it. Typically, though, you do multiple copyright lines somewhere, something like 

> Copyright (C) 2009-2014 Your Mom
> Copyright (c) 2015-2020 Cloudant

etc

Reply:
Hmm, in that case, should I put back those Cloudant copyrights I removed?

Reply:
http://www.apache.org/foundation/license-faq.html#Apply-My-Software

Reply:
Er, sorry, ignore that. https://www.apache.org/legal/src-headers.html#headers

Reply:
Ok, sounds like I should leave those copyrights in, so I've reverted the offending commit.